duneprototypes/Protodune/hd/RawDecoding/PDHDTriggerReader_module.cc

```diff
@@ -17,6 +17,7 @@
 #include "dunecore/DuneObj/RDStatus.h"
 #include "dunecore/DuneObj/DUNEHDF5FileInfo2.h"
 #include "dunecore/HDF5Utils/HDF5RawFile2Service.h"
+#include "daqdataformats/v4_4_0/Fragment.hpp"
```

Comment:
Be careful here -- including the v4_4_0 fragment with DUNEHDF5FileInfo2 mixes old and new formats.  You have included the daqdataformats/v3_4_1/Fragment.hpp and the daqdataformats/v4_4_0/Fragment.hpp headers -- I'm a bit surprised this compiles.